doc/source/theory/fluid_dynamics/fem_formulation.rst

```diff
@@ -0,0 +1,109 @@
+Origin of the finite element formulation
+##########################################
+
+This section describes the FEM formulation used within Lethe. Starting from the strong form of the equations, we obtain describe the weak-form. We then discuss briefly the challenges associated with solving the Navier-Stokes equations before we introduce the two approaches that are available in Lethe to solve them
+
+
+Starting from the incompressible Navier-Stokes equations:
+
+.. math::
+    \partial_l u_l &= 0 
+
+    \partial_t u_k + u_l \partial_l u_k &= -\frac{1}{p} \partial_k p + \nu \partial_l \partial_l u_k + f_k
+
+We consider a domain :math:`\Omega` of contour :math:`\Gamma`. Without loss of generality, we assume Dirichlet boundary conditions or zero stress  conditions 
+on :math:`\Gamma`. We multiply by two test functions :math:`q` and :math:`\mathbf{v}=v_k` for pressure and velocity respectively and integrate over the domain :math:`\Omega`. The resulting set of equation is:
+
+.. math::
+
+  &\int_{\Omega}  q  \partial_l u_l d\Omega =0 
+  \\
+  &\int_{\Omega}  v_k \left(\partial_t u_k+ u_l \partial_l u_k + \partial_k p - \nu \partial_l \partial_l u_k - f_k \right) d\Omega =0
+
+
+Because we want the pressure to be in :math:`\mathcal{L}^2` and the velocity to be in :math:`\mathcal{H}^1`, we integrate by part the viscous stress and the pressure gradient. We thus obtain the weak form:
+
+
+.. math::
+
+  &\int_{\Omega}  q  \partial_l u_l \mathrm{d}\Omega =0 
+  \\
+  &\int_{\Omega}  v_k \left(\partial_t u_k+ u_l \partial_l u_k - \nu \partial_l \partial_l u_k - f_k \right) \mathrm{d}\Omega 
+  \\
+  &  - \int_{\Omega} \left( \partial_k \right) v_k p \mathrm{d}\Omega  
+ + \nu \int_{\Omega} \left( \partial_l v_k \right) \left( \partial_l u_k  \right) \mathrm{d}\Omega  
+ \\
+  &  + \int_{\Gamma} \left( v_k \right) \left( \partial_l u_k  +\delta_{lk} p \right) n_l \mathrm{d}\Gamma
+   =0
+
+where :math:`\delta_{lk}` is the Kronecker delta and :math:`n_j` is the outward poiting normal vector to a surface. Since we assume Dirichlet boundary conditions or zero stress  conditions 
+on :math:`\Gamma`, this term may be discarded. Thus, when no boundary condition is applied, the boundary condition applied is:
+
+.. math::
+
+    \int_{\Gamma} \left( v_k \right) \left( \partial_l u_k  +\delta_{lk} p \right) n_l \mathrm{d}\Gamma=0
+
+which can be seen as an outlet boundary condition where the normal stress is zero. In essence, this can be used to approximatively imposed an outlet boundary condition with a zero average pressure.
```

Comment:
```suggestion
which can be seen as an outlet boundary condition where the normal stress is zero. In essence, this can be used to approximatively impose an outlet boundary condition with a zero average pressure.
```